core/tests/test_utils.py

```diff
@@ -529,6 +625,23 @@ def save_new_valid_collection(
             language_code=constants.DEFAULT_LANGUAGE_CODE,
             exploration_id='an_exploration_id',
             end_state_name=DEFAULT_END_STATE_NAME):
+        """Creates an Oppia collection and adds a node saving the
+        exploration details.
+
+        Args:
+            collection_id: str. Id for the collection to be created.
+            owner_id: str. user_id for owner to be linked to the collection.
+            title: str. title for the collection.
+            category: str. category of the exploration.
+            objective: str. objective for the exploration.
+            language_code: str. The language code for the exploration.
+            exploration_id: str. exploration_id for the Oppia exploration.
+            end_state_name: str. The name of the end state for the exploration.
+
+        Returns:
+            Collection. Newly created collection containing corresponding
```

Comment:
**A newly-created** collection containing **the** corresponding ...

Reply:
Done!